src/main/resources/openapi/schemas/reportTitle.json

```diff
@@ -17,12 +17,12 @@
     },
     "kbTitleId": {
       "type": "string",
-      "description": "KB title identifier",
+      "description": "KB title identifier. If not given, title is ignored (kbManualMatch=true) or unmatched (kbManualMatch=false)",
```

Comment:
Nice to see a JSON Schema `description` containing some actual information! :-)

Reply:
Thx